README.md

```diff
@@ -66,6 +71,7 @@ ytch.getChannelInfo(channelId, channelIdType).then((response) => {
    isVerified: Boolean,
    tags: Array[String], // Will return null if none exist
    channelIdType: Number, 
+   alert: String, // Will return a response alert message if any (e.g., "This channel does not exist."). Otherwise null
```

Comment:
What do you think about naming it as `alertMessage`/`alertMsg`?
I image `alert` being an object with multiple properties, like `error`.